addons/common/functions/fnc_getTargetObject.sqf

```diff
@@ -17,11 +17,9 @@
 
 params ["_maxDistance"];
 
-private _position = ATLToASL positionCameraToWorld [0, 0, 0];
-_position set [2, (_position select 2) - (getTerrainHeightASL _position min 0)];
```

Comment:
I do not understand the purpose of this, which is why I'm hesitant to see it go. Do you have any idea what it's supposed to do?

Reply:
I believe this code is from before we had https://community.bistudio.com/wiki/AGLToASL

So it's attempting to do the same, probably more readable if it was like
```
private _position = ATLToASL positionCameraToWorld [0, 0, 0];
private _tHeight = getTerrainHeightASL _position;
if (_tHeight < 0) then {  // over the ocean
    _position set [2, (_position select 2) - _tHeight];
};
```

but either way has a slightly subtle error in that
> AGL Z is the same as in PositionASLW when over sea

so it was only subtracting (terrain -> Sea Level), but not wave height

you can see the error by standing on a dock/carrier, standing still and checking output
the z (which should be ASL and unchanging) will varry over time based on waves when it should be constant